src/api.rs

```diff
@@ -324,6 +327,12 @@ impl Default for ChromaSamplePosition {
   }
 }
 
+/// Pixel data can be encoded over 8 or 10 (or 12 but unsupported) bits. When data is expressed in limited range
+/// you need to take into account different lower and higher bounds that can be used to express color information:
+///  - 8 bit limited ranges are within 16-235, anything below 16 will be treated as pure black, anything above 235
+///    will be treated as pure white, whereas for 10 bit the same limits apply between 64 - 940.
+///  - Of course when using full range you may use the whole width of the available levels (0-255 for 8 bit,
+///    and 0-1023 for 10 bit) to represent data.
```

Comment:
these amounts are true for luma only, chroma has a different range (16-240)

also it is a bit verbose, this could easily fit in a single sentence: when writing documentation you can usually avoid adverbs and connecting sentences with "of course"